blueoil/templates/lmnet/classification.tpl.py

```diff
@@ -72,7 +73,11 @@
 
 PRE_PROCESSOR = Sequence([
     Resize(size=IMAGE_SIZE),
+{% if quantize_first_convolution %}
```

Comment:
It is slight things, you can ignore.

The generated config python file by `python ./blueoil/generate_lmnet_config.py xx.yaml` has empty line like this:
<img width="329" alt="screen shot 2018-11-13 at 19 56 08" src="https://user-images.githubusercontent.com/138891/48409061-3b349f00-e77e-11e8-9e36-ee635067eefa.png">

`{%-`,  `-%}` will eliminate the empty line.
Plz See the doc of jinja: http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control

Reply:
I already solved this issue.
However, using {%- .... -%} also removes the tab space in front of DivideBy255() 
so I just moved all the if clause into one line.

Reply:
@lm-jira 
Thanks 😄 

Sorry, `{%-` is not needs.
The code be solved, also. 
```
PRE_PROCESSOR = Sequence([
    ResizeWithGtBoxes(size=IMAGE_SIZE),
    {% if quantize_first_convolution -%}
    DivideBy255()
    {% else -%}
    PerImageStandardization()
    {% endif -%}
])
```

Reply:
Removing the `{%-` still remove the tab space at the beginning of the line as shown below.
<img width="233" alt="screen shot 2018-11-14 at 16 19 41" src="https://user-images.githubusercontent.com/43122237/48466391-1a755380-e829-11e8-86d9-2af8c57072ae.png">